bin/ovpn_getclient

```diff
@@ -5,25 +5,42 @@
 #
 
 if [ "$DEBUG" == "1" ]; then
-  set -x
+    set -x
 fi
 
 set -e
 
-source "$OPENVPN/ovpn_env.sh"
-cn=$1
+if [ -z "$OPENVPN" ]; then
```

Comment:
The reason for this is simple. Users (including myself) might run this script outside of docker too.